configs/forseti_conf_server.yaml

```diff
@@ -0,0 +1,392 @@
+global:
+
+    # Remove or comment out the email properties if you do not want Forseti to
+    # send email notifications.
+    email_recipient: ${forseti_email_recipient}
+    email_sender: ${forseti_email_sender}
+    sendgrid_api_key: ${sendgrid_api_key}
+
+##############################################################################
+
+inventory:
+
+    # Root resource to start crawling from, formatted as
+    # <resource_type>/<resource_id>, (e.g. "organizations/12345677890")
+    root_resource_id: ${root_resource_id}
+
+    # gsuite access
+    domain_super_admin_email: ${gsuite_admin_email}
+
+    api_quota:
+        # We are not using the max allowed API quota because we wanted to
+        # include some rooms for retries.
+        # Period is in seconds.
+        admin:
+          max_calls: 14
+          period: 1.0
+        appengine:
+          max_calls: 18
+          period: 1.0
+        bigquery:
+          max_calls: 160
+          period: 1.0
+        cloudbilling:
+          max_calls: 5
+          period: 1.2
+        compute:
+          max_calls: 18
+          period: 1.0
+        container:
+          max_calls: 9
+          period: 1.0
+        crm:
+          max_calls: 4
+          period: 1.2
+        iam:
+          max_calls: 18
+          period: 1.0
+        logging:
+          max_calls: 1
+          period: 1.1
+        securitycenter:
+          max_calls: 1
+          period: 1.1,
+        servicemanagement:
+          max_calls: 2
+          period: 1.1
+        sqladmin:
+          max_calls: 1
+          period: 1.1
+
+    # Number of days to retain inventory data:
+    #  -1 : (default) keep all previous data forever
+    #   0 : delete all previous inventory data before running
+    retention_days: -1
+
+##############################################################################
+
+scanner:
+
+    # Output path (do not include filename).
+    # If GCS location, the format of the path should be:
+    # gs://bucket-name/path/for/output
+    output_path: gs://${storage_bucket_name}/scanner_violations
+
+    # Rules path (do not include filename).
+    # If GCS location, the format of the path should be:
+    # gs://bucket-name/path/for/rules_path
+    # if no rules_path is specified, rules are
+    # searched in /path/to/forseti_security/rules/
+    rules_path: /home/ubuntu/forseti-security/rules
```

Comment:
If users decide to deploy Forseti to a directory other than `/home/ubuntu`, this path will have to change as well.

Reply:
Right.

Reply:
Moved to https://github.com/terraform-google-modules/terraform-google-forseti/issues/17